tests/test_e2e_18_mef_eline.py

```diff
@@ -229,3 +230,93 @@ def test_020_patch_intra_evc(self):
 
         data = response.json()
         assert data["uni_z"]["interface_id"] != "00:00:00:00:00:00:00:02:1"
+
+    def test_030_interface_disabled(self):
+        payload = {
+            "name": "Test EVC",
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+                "tag": {
+                    "tag_type": "vlan",
+                    "value": 100,
+                },
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1",
+                "tag": {
+                    "tag_type": "vlan",
+                    "value": 100,
+                },
+            },
+            "enabled": True,
+            "dynamic_backup_path": True,
+        }
+        api_url = KYTOS_API + "/kytos/mef_eline/v2/evc/"
+        response = requests.post(api_url, json=payload)
+        assert response.status_code == 201, response.text
+        data = response.json()
+
+        evc_id = data["circuit_id"]
+
+        # Disable current uni_a
+        api_url = KYTOS_API + "/kytos/topology/v3/interfaces/00:00:00:00:00:00:00:01:2/disable"
```

Comment:
The test case is good, and we can keep it. But looks like you meant to use uni_a `00:00:00:00:00:00:00:01:1` here, right? and in that case let's also assert that the path won't be removed too, but only EVC status gets deactivated.